src/services/anchor-service.ts

```diff
@@ -221,6 +223,9 @@ export class AnchorService {
     logger.debug('Persisting results to local database')
     const numAnchoredRequests = await this._persistAnchorResult(anchors, candidates)
 
+    // maybe here or somewhere in a loop get the time deltas for the successful anchors TODO
```

Comment:
I am a little confused about what we are trying to time here as well.

Reply:
we'll pull out the timing and put it in a separate PR